Formula/infracost.rb

```diff
@@ -0,0 +1,78 @@
+class Infracost < Formula
+  desc "Cost estimates for Terraform"
+  homepage "https://docs.infracost.io/"
+  url "https://github.com/infracost/infracost/archive/v0.4.1.tar.gz"
+  sha256 "d466a917de411020a828c59b555af3c9de1e1a686888b875e6b82432caf3c5c0"
+  license "Apache-2.0"
+  head "https://github.com/infracost/infracost.git"
+
+  depends_on "go" => :build
+  depends_on "terraform" => :test
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/infracost"
+    pkgshare.install "examples"
+  end
+
+  test do
+    expected = <<~EOS
```

Comment:
The JSON output for the example has been updated to the following (this was from the refactor in v0.5.1 so we could support usage-based resources such as AWS Lambda):
```
[
   {
      "name":"aws_ebs_volume.storage_option_1",
      "hourlyCost":"0.0114726027397260325",
      "monthlyCost":"8.375000000000003725",
      "costComponents":[
         {
            "name":"Storage",
            "unit":"GB-months",
            "hourlyQuantity":"0.0205479452054795",
            "monthlyQuantity":"15",
            "price":"0.125",
            "hourlyCost":"0.0025684931506849375",
            "monthlyCost":"1.875"
         },
         {
            "name":"Storage IOPS",
            "unit":"IOPS-months",
            "hourlyQuantity":"0.136986301369863",
            "monthlyQuantity":"100",
            "price":"0.065",
            "hourlyCost":"0.008904109589041095",
            "monthlyCost":"6.5"
         }
      ]
   },
   {
      "name":"aws_ebs_volume.storage_option_2",
      "hourlyCost":"0.001027397260273975",
      "monthlyCost":"0.75000000000000175",
      "costComponents":[
         {
            "name":"Storage",
            "unit":"GB-months",
            "hourlyQuantity":"0.0205479452054795",
            "monthlyQuantity":"15",
            "price":"0.05",
            "hourlyCost":"0.001027397260273975",
            "monthlyCost":"0.75"
         }
      ]
   },
   {
      "name":"aws_instance.web_app",
      "hourlyCost":"0.01245479452054795",
      "monthlyCost":"9.0920000000000035",
      "costComponents":[
         {
            "name":"Compute (on-demand, t3.micro)",
            "unit":"hours",
            "hourlyQuantity":"1",
            "monthlyQuantity":"730",
            "price":"0.0104",
            "hourlyCost":"0.0104",
            "monthlyCost":"7.592"
         }
      ],
      "subresources":[
         {
            "name":"root_block_device",
            "hourlyCost":"0.00205479452054795",
            "monthlyCost":"1.5000000000000035",
            "costComponents":[
               {
                  "name":"Storage",
                  "unit":"GB-months",
                  "hourlyQuantity":"0.0205479452054795",
                  "monthlyQuantity":"15",
                  "price":"0.1",
                  "hourlyCost":"0.00205479452054795",
                  "monthlyCost":"1.5"
               }
            ]
         }
      ]
   },
   {
      "name":"aws_lambda_function.lambda",
      "hourlyCost":"0",
      "monthlyCost":"0",
      "costComponents":[
         {
            "name":"Duration",
            "unit":"GB-seconds",
            "hourlyQuantity":"0",
            "monthlyQuantity":"0",
            "price":"0.0000166667",
            "hourlyCost":"0",
            "monthlyCost":"0"
         },
         {
            "name":"Requests",
            "unit":"requests",
            "hourlyQuantity":"0",
            "monthlyQuantity":"0",
            "price":"0.0000002",
            "hourlyCost":"0",
            "monthlyCost":"0"
         }
      ]
   },
   {
      "name":"aws_nat_gateway.nat",
      "hourlyCost":"0.045",
      "monthlyCost":"32.85",
      "costComponents":[
         {
            "name":"Per GB data processed",
            "unit":"GB/month",
            "hourlyQuantity":"0",
            "monthlyQuantity":"0",
            "price":"0.045",
            "hourlyCost":"0",
            "monthlyCost":"0"
         },
         {
            "name":"Per NAT Gateway",
            "unit":"hours",
            "hourlyQuantity":"1",
            "monthlyQuantity":"730",
            "price":"0.045",
            "hourlyCost":"0.045",
            "monthlyCost":"32.85"
         }
      ]
   }
]```